static/app/views/settings/project/server-side-sampling/serverSideSampling.tsx

```diff
@@ -451,6 +456,8 @@ export function ServerSideSampling({project}: Props) {
           )}
         />
 
+        <LimitedAvailabilityProgramEndingAlert organization={organization} />
```

Comment:
Out of curiosity, how are `component:` hooks different than `feature-disabled:` hooks?
Should we use `<Feature features={['organizations:dynamic-sampling-basic', 'organizations:dynamic-sampling-advanced']} hookName="feature-disabled:dynamic-sampling-la" />` instead?

Reply:
As far as I un understood the component hook is when we wrap another component?  tbh this is not very clear to me and we do it in many different ways in the code... maybe @dashed  and @markstory can help us here?

Reply:
I copied the logic from here https://github.com/getsentry/sentry/blob/5dfe23b830db6d9150e5d2d7f3b1612fde061d71/static/app/views/organizationStats/index.tsx#L38 😄

Reply:
`HookOrDefault()` is simpler and easier to reason about here.

Reply:
we could maybe add a description to the component...something like: "hey are you wondering if you should use x or y? x is useful when...."